lib/features/appConfiguration/data/entity/remote_app_configuration_dto.dart

```diff
@@ -0,0 +1,21 @@
+import 'package:json_annotation/json_annotation.dart';
+
+/*
+ * Command to rebuild the entity.g.dart file:
```

Comment:
Nit: `Command to rebuild the remote_app_configuration_dto.g.dart file:`

Or maybe just remove that specific file name, it was also hard for me to not forget to change this comment 😅

Reply:
Changed